controller/src/main/java/io/pravega/controller/task/Stream/StreamTransactionMetadataTasks.java

```diff
@@ -323,7 +326,7 @@ public void initializeStreamWriters(final EventStreamWriter<CommitEvent> commitW
                                                                                    final OperationContext ctx) {
         // Step 1. Validate parameters.
         CompletableFuture<Void> validate = validate(lease);
```

Comment:
TimeoutServiceConfig#maxlease also needs to be updated from the current 30seconds?

Reply:
TimeoutServiceConfig is set to config.max_lease_value in Main.java
```
            TimeoutServiceConfig timeoutServiceConfig = TimeoutServiceConfig.builder()
                    .maxLeaseValue(Config.MAX_LEASE_VALUE)
                    .build();
```
So we dont have a separate configuration for it from the txn max lease.